### PR TITLE
[enhancement] Added more excuses to AutoExcuse

### PR DESCRIPTION
**Pull request by natan515**
I didn't do anything useful yet lol, but I'm learning Kotlin rn so I hope I can actually code later.

**Describe the pull**
Just added some more auto excuses and clients.

**Describe how this pull is helpful**
Nice to have some more sentences cus rn there's only like 7, I know there an option to add more but more ppl don't use it.

**Additional context**
Nothing.

### CODE REVIEW

File: src/main/java/me/zeroeightsix/kami/module/modules/chat/AutoExcuse.kt [hunk 1]
```diff
@@ -35,7 +35,20 @@ object AutoExcuse : Module() {
             "Wow, so many try hards",
             "Lagggg",
             "I wasn't trying",
-            "I'm not using $CLIENT_NAME client"
+            "I'm not using $CLIENT_NAME client",
+            "Thers to much lag",
+            "My dog ate my pc",
+            "Sorry, %CLIENT_NAME Client is really bad",
```
Comment (l1ving) [inline src/main/java/me/zeroeightsix/kami/module/modules/chat/AutoExcuse.kt:41]: `%` is not detected here, use `$`

File: src/main/java/me/zeroeightsix/kami/module/modules/chat/AutoExcuse.kt [hunk 2]
```diff
@@ -45,6 +58,9 @@ object AutoExcuse : Module() {
             "Future",
             "Salhack",
             "Pyro",
+            "Xulo",
+            "Phobos",
```
Comment (l1ving) [inline src/main/java/me/zeroeightsix/kami/module/modules/chat/AutoExcuse.kt:62]: remove this, nobody knows what it is

File: src/main/java/me/zeroeightsix/kami/module/modules/chat/AutoExcuse.kt [hunk 3]
```diff
@@ -45,6 +58,9 @@ object AutoExcuse : Module() {
             "Future",
             "Salhack",
             "Pyro",
+            "Xulo",
```
Comment (l1ving) [inline src/main/java/me/zeroeightsix/kami/module/modules/chat/AutoExcuse.kt:61]: literally nobody has heard of this before

**Review by l1ving — CHANGES_REQUESTED** [review]

File: src/main/java/me/zeroeightsix/kami/module/modules/chat/AutoExcuse.kt (hunk 1 above)
Comment (natan515) [reply src/main/java/me/zeroeightsix/kami/module/modules/chat/AutoExcuse.kt:41]: Ye sorry, idk why I just press 5 instead of 4.

**Review by l1ving — APPROVED** [review]

**Comment by natan515** [comment]
Rusherhack client sounds weird

**Pull request merged**